velox/exec/prefixsort/tests/utils/EncoderTestUtils.cpp

```diff
@@ -30,9 +30,9 @@ void decodeNoNulls(char* encoded, int64_t& value) {
 } // namespace
 
 void encodeInPlace(std::vector<int64_t>& data) {
+  const static auto encoder = PrefixSortEncoder(true, true);
   for (auto i = 0; i < data.size(); i++) {
-    PrefixSortEncoder::encode(
-        data[i], (char*)data.data() + i * sizeof(int64_t));
+    encoder.encodeNoNulls(data[i], (char*)data.data() + i * sizeof(int64_t));
```

Comment:
I noticed that we currently assume that the size of encoded data equals the size of non-encoded data + 1 (if nullable). It might be helpful to add some APIs to the encoder to allow callers to find out the size of to-be-encoded data instead of assuming it.

Reply:
>  add some APIs to the encoder to allow callers to find out the size of

Is it reasonable to add some constants such as kTimestampEncodedSize to the Encoder?

Reply:
I'm not sure yet. Need to see more code using the encoder.